src/BugsnagUnity/Delivery.cs

```diff
@@ -33,6 +33,19 @@ class Delivery
 
         private const string STRING_TRUNCATION_MESSAGE = "***{0} CHARS TRUNCATED***";
 
+        private const string BREADCRUMB_TRUNCATION_MESSAGE = "Removed, along with {0} older breadcrumbs, to reduce payload size";
+
+        private const string EVENT_KEY = "event";
+
+        private const string BREADCRUMBS_KEY = "breadcrumbs";
+
+        private const string METADATA_KEY = "metaData";
+
+        private const string TYPE_KEY = "type";
+
+        private const string NAME_KEY = "name";
```

Comment:
Just querying if there's any better home for these constants?

I would also suggest naming them a little more clearly so they say what they do rather than describe the content of the string:

```suggestion
        private const string EVENT_KEY_EVENT = "event";

        private const string EVENT_KEY_BREADCRUMBS = "breadcrumbs";

        private const string EVENT_KEY_METADATA = "metaData";

        private const string EVENT_KEY_BREADCRUMB_TYPE = "type";

        private const string EVENT_KEY_BREADCRUMB_MESSAGE = "name";
        
        private const string EVENT_KEY_BREADCRUMB_METADATA = "metaData";
```

Reply:
Renamed 👍 We could consider a BugsnagConstants class at some point